src/kvstore/raftex/SnapshotManager.cpp

```diff
@@ -71,34 +71,34 @@ folly::Future<Status> SnapshotManager::sendSnapshot(std::shared_ptr<RaftPart> pa
                                                       << ", totalSize " << totalSize;
                             p.setValue(Status::OK());
                         }
-                        return;
+                        return true;
                     }
                 } catch (const std::exception& e) {
                     LOG(ERROR) << part->idStr_ << "Send snapshot failed, exception " << e.what();
                     p.setValue(Status::Error("Send snapshot failed!"));
-                    return;
+                    return false;
```

Comment:
Do we need to return false when we fail only once? Or you think the timeout is big enough?

Reply:
Make sense.  Maybe we should add a timeout control and retry when catching exceptions.